Sources/Auth/Auth.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import WalletConnectRelay
+import Combine
+
+public class Auth {
+
+    public static var instance: AuthClient = {
+        guard let metadata = Auth.metadata else {
+            fatalError("Error - you must call Sign.configure(_:) before accessing the shared instance.")
+        }
+        return AuthClientFactory.create(
+            metadata: metadata,
+            account: account,
+            relayClient: Relay.instance)
+    }()
+
+    private static var metadata: AppMetadata?
+    private static var account: Account?
+
+    private init() { }
+
+    static public func configure(metadata: AppMetadata, account: Account?) {
+        Auth.metadata = metadata
```

Comment:
mb config if more than 1 property?

Reply:
relay have more and we do not use config.
you removed the Config and actually I liked it